src/main/java/com/myhome/helper/CommonHelper.java

```diff
@@ -0,0 +1,49 @@
+package com.myhome.helper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class CommonHelper {
```

Comment:
This class is really unnecessary

Reply:
I think there should be a class with common utility functions..??

Reply:
Do we really need a helper to check emptiness?

Reply:
No we don't. But that function can be removed. But other functions like adding number of days to a date, date fomratting, converting an object to json string this can be done at one place..
That is why I created the class..

Reply:
It is better to rename this class to do one specific thing and avoid having any sort of utility class.

Reply:
After looking at the entire set of changes, It is better to remove this class altogether.